lib/stimulus_reflex/channel.rb

```diff
@@ -100,8 +101,15 @@ def render_page(url, reflex)
   def broadcast_morphs(selectors, data, html)
     document = Nokogiri::HTML(html)
     selectors.each do |selector|
-      html = document.css(selector).inner_html
-      cable_ready[stream_name].morph selector: selector, html: html, children_only: true, permanent_attribute_name: "data-reflex-permanent", stimulus_reflex: data
+      match = document.css(selector)
+      next if match.blank?
+      cable_ready[stream_name].morph(
+        selector: selector,
+        html: match.inner_html,
+        children_only: true,
+        permanent_attribute_name: "data-reflex-permanent",
```

Comment:
Why did you go with data-reflex-permanent instead of data-reflex-ignore?

It's shorter, and I don't have to mentally force myself not to type permenent every damn time. :)

Reply:
I wanted familiarity and consistency for Rails developers. See: data-turonolinks-permanent.  https://github.com/turbolinks/turbolinks#persisting-elements-across-page-loads